src/Inventory.cpp

```diff
@@ -78,7 +78,7 @@ int cInventory::HowManyCanFit(const cItem & a_ItemStack, int a_BeginSlotNum, int
 	}
 
 	char NumLeft = a_ItemStack.m_ItemCount;
-	int MaxStack = ItemHandler(a_ItemStack.m_ItemType)->GetMaxStackSize();
+	int MaxStack = cItemHandler::For(a_ItemStack.m_ItemType).GetMaxStackSize();
```

Comment:
Similarly here, GetMaxStackSize exists directly on cItem, we should either be using these helpers, or getting rid of them.

Reply:
I am just using the helpers